lldb/packages/Python/lldbsuite/test/lang/c/modules/main.c

```diff
@@ -5,16 +5,16 @@ int printf(const char * __restrict format, ...);
 typedef struct {
     int a;
     int b;
-} MYFILE;
+} FILE;
 
 int main()
 {
-    MYFILE *myFile = malloc(sizeof(MYFILE));
+  FILE *myFile = malloc(sizeof(FILE));
 
-    myFile->a = 5;
-    myFile->b = 9;
+  myFile->a = 5;
+  myFile->b = 9;
 
-    printf("%d\n", myFile->a + myFile->b); // Set breakpoint 0 here.
+  printf("%d\n", myFile->a + myFile->b); // Set breakpoint 0 here.
 
-    free(myFile);
+  free(myFile);
 }
```

Comment:
I suppose this is from the revert of the AST commit?

Reply:
Yeah I think so. `clang-format` was complaining at least.

Reply:
We do not want to modify generic stuff because of ``clang-format`` I believe

Reply:
Reverted